example/dash-odd-links/index.html

```diff
@@ -28,16 +29,8 @@
       .graphData(gData)
       .nodeRelSize(8)
       .linkWidth(3)
-      .linkCanvasObjectMode('highlight')
-      .linkCanvasObject((link, ctx) => {
-        ctx.setLineDash([5, 5]);
-        ctx.lineWidth = 3;
-        ctx.strokeStyle = '#ccc';
-        ctx.moveTo(link.source.x, link.source.y);
-        ctx.lineTo(link.target.x, link.target.y);
-
-        // paint all on last link
-        (link === gData.links[gData.links.length - 1]) && ctx.stroke();
-      });
+      .linkLineDash(link => link.dashed === true ? [5, 5] : []) //Variant 1
+      //.linkLineDash("dashed")                                   //Variant 2
```

Comment:
No need to maintain the other variants as comments in the example. Cleaner without, since it's just syntactic variation.

Reply:
Removed commented variants